docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -2060,6 +2060,8 @@ This section contains reference information about each of the configuration prop
 |`fhirServer/bulkdata/core/maxInputs`|number| The number of inputs allowed for $import |
 |`fhirServer/bulkdata/core/iamEndpoint`|string| Override the system's IAM endpoint |
 |`fhirServer/bulkdata/core/maxChunkReadTime`|string| Max time in milliseconds to read during a bulkdata export without type filters. The time should be three quarters of the transactionManager timeout (often the FHIR_TRANSACTION_MANAGER_TIMEOUT value). Note, this value is a string representation of a long value.|
+|`fhirServer/bulkdata/core/defaultExportProvider`|string| The default Export storage provider used by the Bulk Data Import and Export|
+|`fhirServer/bulkdata/core/defaultImportProvider`|string| The default Import storage provider used by the Bulk Data Import and Export|
```

Comment:
can we also add these to 5.1.1 to indicate that the default value for these settings is "default" ?

Reply:
Updated